sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/EnsureRequirements.scala

```diff
@@ -54,7 +54,7 @@ case class EnsureRequirements(
     assert(requiredChildDistributions.length == originalChildren.length)
     assert(requiredChildOrderings.length == originalChildren.length)
     // Ensure that the operator's children satisfy their output distribution requirements.
-    var newChildren = originalChildren.zip(requiredChildDistributions).map {
+    var children = originalChildren.zip(requiredChildDistributions).map {
```

Comment:
I feel the original name `children` is better, as it's a `var` and we keep updating it.